core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -143,6 +143,9 @@ THE SOFTWARE.
     <l:yui module="storage" />
     <!--l:yui module="editor" suffix="-beta" /-->
 
+    <!-- closest polyfill for IE 11 -->
```

Comment:
given the more generic filename now, this comment looks like it would quickly be outdated.

Reply:
```suggestion
```